sycl/test-e2e/AOT/half.cpp

```diff
@@ -0,0 +1,31 @@
+// This test ensures that a program that has a kernel
+// using fp16 can be compiled AOT.
+
+// REQUIRES: ocloc, opencl-aot
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_tgllp -o %t.tgllp.out %s
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_cfl -o %t.cfl.out %s
```

Comment:
Can you please apply a similar fix that I have proposed in https://github.com/intel/llvm/pull/14588, as written this will fail on Windows once I enable windows aot.

Reply:
[e328ae7](https://github.com/intel/llvm/pull/14590/commits/e328ae7634854ecda36bd5417169886b2db782b7)